ocpp/v16/call.py

```diff
@@ -229,3 +234,72 @@ class DataTransferPayload:
     vendor_id: str
     message_id: Optional[str] = None
     data: Optional[str] = None
+
+
+# Backported CALL Messages
+# Improved security for OCPP 1.6-J specification
+
+@dataclass
+class CertificateSignedPayload:
+    cert: List
+
+
+@dataclass
+class DeleteCertificatePayload:
+    certificate_hash_data: Dict
+
+
+@dataclass
+class ExtendedTriggerMessagePayload:
+    requested_message: MessageTriggerType
+
+
+@dataclass
+class GetInstalledCertificateIdsPayload:
+    type_of_certificate: CertificateUseType
```

Comment:
Please use the field name `certificate_type`.

![Uploading Screenshot_2021-04-26_11-20-00.png…]()